test/library/standard/Map/mapErrors.chpl

```diff
@@ -0,0 +1,23 @@
+use Map;
+
+var m = new map(int, int);
+
+m[0] = 1;
+
+m.getValue(0);
+
+var caughtVal: int;
+var sentinelVal: int;
+
+try {
+  m.getValue(1);
+} catch e {
+  caughtVal = 1;
+  writeln("caught error");
```

Comment:
Could you either make `e` non generic, or print its type in the `catch` block to make sure that we are catching `KeyNotFoundError` and not something else here?